app/src/main/java/com/cortado/minionify/accumulators/Preferences.java

```diff
@@ -0,0 +1,5 @@
+package com.cortado.minionify.accumulators;
+
+public class Preferences {
+    public static final String APP_MODE_PREFERENCE_KEY = "mode";
```

Comment:
you can rename to `APP_MODE_KEY` the context is inferred

Reply:
Done.